noir-projects/aztec-nr/aztec/src/utils/bytes.nr

```diff
@@ -0,0 +1,206 @@
+// Converts the input bytes into an array of fields. A Field is ~254 bits meaning that each field can store 31 bytes.
+// This implies that M = ceil(N / 31).
+pub fn bytes_to_fields<let N: u32, let M: u32>(input: [u8; N]) -> [Field; M] {
```

Comment:
What do you think about creating new traits as discussed in https://github.com/AztecProtocol/aztec-packages/issues/7728 and making these fns the impl for u8 arrays?